### PR TITLE
Setup nexus param order

### PR DESCRIPTION
**Pull request by rustydb**
### Summary and Scope

Relates to: MTL-2237

#### Issue Type

- Bugfix Pull Request
- Docs Pull Request
- RFE Pull Request

Changes the documentation to use the new Fawkes tarball.

Includes several other fixes from changes to fawkes since the last production run:
- Numerous UX documentation fixes (see commits)
- Inclusion of `terragrunt`
- Gitea fixes
- `crucible` usage fixes
- MTL-2302

See commits for detailed logs.

This also depends on:
- https://github.com/Cray-HPE/crucible/pull/24
- https://github.com/Cray-HPE/metal-provision/pull/562
- https://github.com/Cray-HPE/fawkes-terraform/pull/15

### Prerequisites

- [ ] I have included documentation in my PR (or it is not required)
- [x] I tested this on internal system (if yes, please include results or a description of the test)
- [ ] I tested this on a vshasta system (if yes, please include results or a description of the test)
 
### Risks and Mitigations
 

This is the result from running through our fawkes deployment on redbull.

### CODE REVIEW

File: modules/installation/pages/1-preface/2-installation-requirements.adoc [hunk 1]
```diff
@@ -6,7 +6,7 @@ For the Fawkes installation, one will need:
 - A workstation with access to the first server to use as the hypervisor
 - A Fawkes tar ball
 +
-NOTE: At this time, there is no Fawkes tar ball. Artifacts are downloaded in the xref:usb-creation.adoc[] stage.
+NOTE: At this time, there is no Fawkes tar ball. Artifacts are downloaded in the xref:2-pre-installation/1-usb-creation.adoc[]] stage.
```
Comment (manderson-hpe) [inline modules/installation/pages/1-preface/2-installation-requirements.adoc]: "tarball"

**Review by manderson-hpe — APPROVED** [review]

File: modules/ROOT/pages/services/nexus/index.adoc [hunk 2]
```diff
@@ -0,0 +1,11 @@
+= Nexus
+:toc:
+:toclevels: 3
+
+This module provides operational guides for Fawke's intance of link:https://www.sonatype.com/products/sonatype-nexus-repository[Sonatype Nexus].
```
Comment (jpdavis-prof) [inline modules/ROOT/pages/services/nexus/index.adoc]: s/intance/instance/ ?

File: modules/ROOT/pages/services/nexus/http-proxy.adoc [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
++
+image::nexus/nexus-signin.png[Signing in,align="left"]
+. Click the "cog" at the top of the page
++
+image::nexus/nexus-cog.png[Server configuration,align="left"]
+. Find "HTTP" on the left underneath "System"
++
+image::nexus/nexus-proxy.png[Proxy configuration,align="left"]
+. Place a check in either HTTP proxy, HTTPS proxy, or both. If the proxy IP/URL given by the network administrator is `http`, check _both_ HTTP and HTTPS proxy and fill them in the same way.
+. Fill in the URL and Port for the proxy for each proxy that was enabled.
```
Comment (jpdavis-prof) [inline modules/ROOT/pages/services/nexus/http-proxy.adoc]: Maybe:
"Fill in the URL and port for each proxy you wish to enable."?

File: modules/installation/pages/1-preface/2-installation-requirements.adoc [hunk 4]
```diff
@@ -4,9 +4,9 @@ For the Fawkes installation, one will need:
 
 - An SSH public key, this will be used to access the management VM
```
Comment (jpdavis-prof) [inline modules/installation/pages/1-preface/2-installation-requirements.adoc]: - SSH key pair

File: modules/installation/pages/1-preface/2-installation-requirements.adoc [hunk 5]
```diff
@@ -4,9 +4,9 @@ For the Fawkes installation, one will need:
...
 - A workstation with access to the first server to use as the hypervisor
```
Comment (jpdavis-prof) [inline modules/installation/pages/1-preface/2-installation-requirements.adoc]: A workstation with access to the first server to use as a hypervisor

File: modules/installation/pages/1-preface/2-installation-requirements.adoc [hunk 6]
```diff
@@ -4,9 +4,9 @@ For the Fawkes installation, one will need:
...
-- A Fawkes tar ball
+- A Fawkes tarball
 +
-NOTE: At this time, there is no Fawkes tar ball. Artifacts are downloaded in the xref:usb-creation.adoc[] stage.
+NOTE: At this time, there is no Fawkes tarball. Artifacts are downloaded in the xref:2-pre-installation/1-usb-creation.adoc[]] stage.
```
Comment (jpdavis-prof) [inline modules/installation/pages/1-preface/2-installation-requirements.adoc]: One too many closing braces.
**1-usb-creation.adoc[]]**

File: modules/installation/pages/2-pre-installation/1-usb-creation.adoc [hunk 7]
```diff
@@ -0,0 +1,186 @@
...
+
+This segment will walk the user through creating the USB with the downloaded tarball.
+
+=== Linux
+
+The Linux method will enable persistence as well as data storage for bringing artifacts into the booted LiveCD.
+
+==== Prerequisites
+
+Prerequisites for creating the USB from a Linux box.
```
Comment (jpdavis-prof) [inline modules/installation/pages/2-pre-installation/1-usb-creation.adoc]: Prerequisites for creating the bootable USB drive from a Linux box

File: modules/installation/pages/2-pre-installation/1-usb-creation.adoc [hunk 8]
```diff
@@ -0,0 +1,186 @@
...
+[source,bash]
+----
+for image_dir in fawkes-${RELEASE}/images/*; do
+    (
+    cd "$image_dir"
+    sha256sum -c ./"$(basename "$image_dir")"*sha256.txt
+    )
+done
+----
+All the images should return with `OK`, if any fail, please stop and open a bug (use the bug opening tab on the bottom right of this screen).
```
Comment (jpdavis-prof) [inline modules/installation/pages/2-pre-installation/1-usb-creation.adoc]: All of the images should return with `OK`. If any fail, please stop and open a bug by using the bug opening tab on the bottom right of this screen.

File: modules/installation/pages/2-pre-installation/1-usb-creation.adoc [hunk 9]
```diff
@@ -0,0 +1,186 @@
...
++
+NOTE: `crucible` is undergoing source code changes and will have packages for `yum` and `debian` package managers in the near future.
++
+* SUSE
++
+[source,bash]
+----
+zypper --plus-repo "fawkes-${RELEASE}/rpm/sle-$(awk -F= '/VERSION=/{gsub(/["-]/, "") ; print tolower($NF)}' /etc/os-release)" in -n crucible
+----
+. Find the USB to use.
```
Comment (jpdavis-prof) [inline modules/installation/pages/2-pre-installation/1-usb-creation.adoc]: Find the desired USB drive.

File: modules/installation/pages/2-pre-installation/1-usb-creation.adoc [hunk 10]
```diff
@@ -0,0 +1,186 @@
...
+. Mount the USB and copy over our downloaded artifacts.
++
+[source,bash]
+----
+sudo mkdir -p /mnt
+sudo mount "${USB}4" /mnt
+sudo cp -p "fawkes-${RELEASE}.tar.gz" /mnt/
+sudo umount /mnt
+----
+. Eject the USB and plug it into the server for deployment.
```
Comment (jpdavis-prof) [inline modules/installation/pages/2-pre-installation/1-usb-creation.adoc]: Eject the USB drive and plug it into the server for deployment.

File: modules/installation/pages/2-pre-installation/1-usb-creation.adoc [hunk 11]
```diff
@@ -0,0 +1,186 @@
...
+
+==== Steps
+
+The following steps will create a bootable USB from a MacOS or Windows machine.
+
+NOTE: `dd` can also be used for macOS users.
+
+. Open belanaEtcher
+. Select the ISO file, the USB stick, and click Flash
+. Eject the USB and insert it into the server
```
Comment (jpdavis-prof) [inline modules/installation/pages/2-pre-installation/1-usb-creation.adoc]: Eject the USB drive and insert it into the server

File: modules/installation/pages/2-pre-installation/1-usb-creation.adoc [hunk 12]
```diff
@@ -0,0 +1,186 @@
+= USB Creation
+:toc:
+:toclevels: 2
+
+This guide walks the user through setting up a bootable USB for bootstrapping the Fawkes cluster.
```
Comment (jpdavis-prof) [inline modules/installation/pages/2-pre-installation/1-usb-creation.adoc:5]: This guide walks the user through setting up a bootable USB drive for bootstrapping the Fawkes cluster.

File: modules/installation/pages/2-pre-installation/3-initial-configuration.adoc [hunk 13]
```diff
@@ -2,6 +2,13 @@
 :toc:
 :toclevels: 3
 
+This section will configure a brand new hypervisor image; first login, and network setup.
```
Comment (jpdavis-prof) [inline modules/installation/pages/2-pre-installation/3-initial-configuration.adoc:5]: This section will configure a new hypervisor installation; first login, and network setup.

Comment (jpdavis-prof) [reply modules/installation/pages/2-pre-installation/3-initial-configuration.adoc:5]: All occurrences of "the USB" should be replaced with something like "the USB drive".

**Review by jpdavis-prof — APPROVED** [review]

**Pull request merged**